neuralforecast/core.py

```diff
@@ -319,6 +320,8 @@ def cross_validation(
             Length of test size. If passed, set `n_windows=None`.
         sort_df : bool (default=True)
             Sort `df` before fitting.
+        fit_models: bool (default=True)
```

Comment:
I think that the description of the arguments function can improve.
to `bool=False, Whether or not we fit models during cross-validation.`